File: 250620/상품 코드/product-code.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String id2 = sc.next();
        int code2 = sc.nextInt();
        // Please write your code here.

        Product p1 = new Product("codetree", 50);
        Product p2 = new Product(id2, code2);

        System.out.printf("product %d is %s\nproduct %d is %s", p1.code, p1.name, p2.code, p2.name);
    }
}

class Product {
    String name;
    int code;

    Product() {
        this.name = "";
        this.code = 0;
    }

    Product(String name, int code) {
        this.name = name;
        this.code = code;
    }
}

File: 250923/상품 코드/product-code.py
product_name, product_code = input().split()
product_code = int(product_code)

# Please write your code here.

class S:
    def __init__(self,product_name,product_code) :
        self.product_name = product_name
        self.product_code = product_code

s1 = S("codetree",50)
s2=S(product_name,product_code)

print(f"product {s1.product_name} is {s1.product_code}")
print(f"product {s2.product_name} is {s2.product_code}")